python2/test_typing.py

```diff
@@ -1896,6 +1898,16 @@ def foo(x):
 
         self.assertIsNone(typing.get_type_hints(foo))
 
+    def test_typing_compiles_with_opt(self):
+        file_path = os.path.join(os.path.dirname(os.path.realpath(__file__)),
+                                 'typing.py')
+        try:
+            subprocess.check_output('python -OO {}'.format(file_path),
+                                    stderr=subprocess.STDOUT,
+                                    shell=True)
+        except:
```

Comment:
Bare except is bad style. It cat catch something unrelated, I was talking about only removing `as e` part, since it was not used.

Reply:
Indeed.  I will add it back immediately.